docs/releases/temporary-versions/version-v12.0.0b1.md

```diff
@@ -0,0 +1,184 @@
+
+The extension provides an easy-to-use API for creating and managing banners, and supports various types of banners, including fixed, floating, and dismissible banners. It also allows banners to be customized with different colors, icons, and animations.
+
+![](./v12.0.0/Banners.png)
+
+### Static pages - administration panel
+
+Another notable addition is the integration of invenio-pages administration panel, a powerful and flexible extension for creating and managing static pages in your InvenioRDM instance. With invenio-pages, you can easily create custom pages for your application, such as About Us, Contact Us, or Terms of Service pages.
+
+Some of the key advantages of using invenio-pages to manage static pages include:
+
+- Easy customization: With invenio-pages, you can quickly and easily create custom pages using a simple, user-friendly interface. You can customize the layout, formatting, and content of your pages, and even add custom CSS and JavaScript code.
+
+- Version control: Invenio-pages includes built-in version control, so you can easily track changes to your pages and roll back to previous versions if needed - this feature is available only for developers
+
+
+### File uploads and restricted records
+
+
+#### Metadata-only 
+
+InvenoRDM v12.0.0b1 introduces two new configuration variables to control whether metadata-only records and restricted files of a record are allowed in the system.
+
+A metadata-only record is a record that contains only descriptive metadata, without any associated files or content. These types of records can be useful in cases where you want to provide metadata for a resource that does not have a corresponding digital object or content file.
+
+By default, InvenioRDM allows metadata-only records. However, with the addition of the RDM_ALLOW_METADATA_ONLY_RECORDS configuration variable, you can now choose to disallow metadata-only records if desired, and any attempt to create or update a record without files or content will result in an error.
+
+To use this configuration variable, add it to your `invenio.cfg`:
+
+```python
+RDM_ALLOW_METADATA_ONLY_RECORDS = False
+```
+
+With this configuration option set to False, users cannot anymore create metadata-only records by omitting the files field when creating or updating a record.
+
+It's worth noting that allowing metadata-only records may have implications for data management and preservation, as well as for editing existing records (users will not be required to provide a file for each record. Therefore, it's recommended to carefully consider the use case and its potential consequences, before disabling this configuration option.
+
+#### Restricted records
+
+A restricted record is a record that contains sensitive or confidential information, and is not publicly accessible. Restricted records can be useful in cases where you want to provide access to certain information or resources to only a select group of users or organizations.
```

Comment:
will we document this as you did for export formats?

Reply:
Maybe it just belongs to the list of "features" you mentioned above that need to be documented as "tutorial" like cases...TLDR: can be taken later indeed!